### PR TITLE
Implement SPP to Thompson MP

### PR DESCRIPTION
**Pull request by bluefinweiwei**
modified:   module_mp_thompson.F90	
        modified:   mp_thompson.F90
	modified:   mp_thompson.meta

Hi Laurie - can you please review these three files with spp for MP? Thanks!

### CODE REVIEW

File: physics/mp_thompson.F90 [hunk 1]
```diff
@@ -594,10 +628,10 @@ subroutine mp_thompson_run(ncol, nlev, con_g, con_rd,        &
                               diagflag=diagflag, do_radar_ref=do_radar_ref_mp,               &
                               re_cloud=re_cloud, re_ice=re_ice, re_snow=re_snow,             &
                               has_reqc=has_reqc, has_reqi=has_reqi, has_reqs=has_reqs,       &
-                              rand_perturb_on=rand_perturb_on, kme_stoch=kme_stoch,          &
+                              rand_perturb_on=spp_mp, kme_stoch=kme_stoch,                   &
```
Comment (llpcarson) [inline physics/mp_thompson.F90:631]: Weiwei - do you know if this is a namelist parameter/setting (the spp_mp flag)?  Or, how it is set?

**Comment by llpcarson** [comment]
These changes look good on the ccpp-physics side.  Question about the spp_mp flag - how is that set?  If it's namelist, we'll need to add it to the FV3 side along with the other changes

File: physics/mp_thompson.F90 (hunk 1 above)
Comment (bluefinweiwei) [reply physics/mp_thompson.F90:631]: Thanks Laurie for reviewing them! Yes, I think this is a namelist parameter to let people choose which variable they want to add the perturbation. Please feel free to check Line 1188-1212 in module_mp_thompson.F90. I've added this to the GFS_layer and submitted a PR for your review.

Comment (bluefinweiwei) [reply physics/mp_thompson.F90:631]: spp_mp has been set as 7 in this code and passed to mp_thompson_run.

**Pull request merged**